src/screens/TopicMemberScreen/index.js

```diff
@@ -135,6 +149,10 @@ const TopicMemberScreen = () => {
     setHeaderHide(false);
   };
 
+  React.useEffect(() => {
+    showAnimationHeader();
+  }, [coverPath]);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new `useEffect` hook is added to call `showAnimationHeader` whenever `coverPath` changes. This seems like a good way to ensure that the header animation is updated when a new cover image is loaded.
```diff
+ React.useEffect(() => {
+   showAnimationHeader();
+ }, [coverPath]);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->